go.mod

```diff
@@ -74,7 +74,7 @@ require (
 	github.com/hashicorp/vault-plugin-auth-azure v0.5.6
 	github.com/hashicorp/vault-plugin-auth-centrify v0.5.5
 	github.com/hashicorp/vault-plugin-auth-cf v0.5.4
-	github.com/hashicorp/vault-plugin-auth-gcp v0.7.0
+	github.com/hashicorp/vault-plugin-auth-gcp v0.7.1-0.20200721115240-07ff53341dfe
```

Comment:
Should this have the same pseudo-version tag as the one on the [1.5.x backport PR](https://github.com/hashicorp/vault/pull/9831/files#diff-37aff102a57d3d7b797f152915a6dc16R72)?